contracts/NodeDelegator.sol

```diff
@@ -342,6 +361,41 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
         ISSVNetwork(SSV_NETWORK_ADDRESS).registerValidator(publicKey, operatorIds, sharesData, amount, cluster);
     }
 
+    /// @dev Exit a validator from the Beacon chain.
+    /// The staked ETH will be sent to the EigenPod.
+    function exitSsvValidator(
+        bytes calldata publicKey,
+        uint64[] calldata operatorIds
+    )
+        external
+        onlyLRTOperator
+        whenNotPaused
+    {
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).exitValidator(publicKey, operatorIds);
+
+        // There can be a gap between the validator exiting the beacon chain and the ETH being sent to the EigenPod.
```

Comment:
After the exit message is signed the validator still needs to perform attestations and is subject to slashing and also has the ETH staked intact. Once the ETH is marked as withdrawable it will be swept by the automatic process. A

It is not entirely correct to already deduct the 32 ETH at this point. Though the sweeping happens on the consensus chain (beacon chain) and is not detectable as an event on the execution chain. Which brings up a question how to effectively account for this, to not double count the ETH or count too little of it

Reply:
Agreed that we can't just sub 32 when we request the withdraw process to begin.

This also means that I don't think we can directly include the EigenPod eth balance in our balance calculations, since it is going to change outside our control, and as a result of funds that we have already accounted for elsewhere.

For example, if we initiate a withdraw and it happens, there's no guarantee that we would be the ones reporting the proofs to the EigenPod and changing the EigenPod's internal accounting - since that's a publicly callable method by anyone.

Reply:
If we have no withdraws, and no reward tracking, we could have a deposit only system that works off the stored value of the original balances.

Reply:
Thanks @sparrowDom. I'm not sure how we'll handle this when we support withdraws but for initial launch of restaking we can drop the validator withdrawal methods. I've done this on a separate PR https://github.com/oplabs/primestaked-eth/pull/54.

The update withdrawal process looks like this
![primeSSVRestakingProcess-withdraw](https://github.com/oplabs/primestaked-eth/assets/6491112/e0992bd6-4765-4c0a-b8bb-c1a015c60849)